src/client/actions/audioactions.js

```diff
@@ -0,0 +1,8 @@
+import { CREATE_AUDIO_DATA, PLAY_PAUSE_SOUND_FROM_FILE, CREATE_STREAME_DATA, START_MUTE_STREAME_AUDIO, MERGE_CANVAS_WIDTH } from './types';
```

Comment:
Rename please this file by using camel case

Reply:
ok